todo/src/components/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import { addTodo, deleteTodo, toggleTodo } from "../actions";
+
```

Comment:
you hit the MVP but I liked how you had the app.js file clean and without as much code in it. But this does work and you know I feel that 'done is done' so good job